ocs_ci/ocs/pv.py

```diff
@@ -0,0 +1,135 @@
+import logging
```

Comment:
I think that the file should reside under ocs_ci/ocs/resources, to keep consistent with other resources

Reply:
Fixed